hedyorg/hedy
# [CHORE] Script to sort all yaml files in the content dir

**Description**

Adds a script to rewrite the yaml files in the content directory deterministically. Like we do with python code using autopep8 but for the translation texts.

Fixes #4541 

**How to test**

- Enable the script in .pre-commit-config.json  and run `pre-commit run --all`.
- Check if unit tests succeed by running them locally on the rewritten yaml files
- Verify if indented parsons still work by going to level 8 -> puzzle and seeing that `print 'You are selected in team... '` is indented.

**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)
